apps/nextjs-example/package.json

```diff
@@ -11,25 +11,27 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@aptos-labs/wallet-adapter-core": "workspace:*",
     "@aptos-labs/wallet-adapter-ant-design": "workspace:*",
     "@aptos-labs/wallet-adapter-mui-design": "workspace:*",
     "@aptos-labs/wallet-adapter-react": "workspace:*",
-    "@blocto/aptos-wallet-adapter-plugin": "^0.1.2",
+    "@blocto/aptos-wallet-adapter-plugin": "^0.2.2",
     "@flipperplatform/wallet-adapter-plugin": "^0.1.0",
     "@haechi-labs/face-aptos-adapter-plugin": "^0.0.4",
-    "@haechi-labs/face-sdk": "^1.10.4",
-    "@haechi-labs/face-types": "^1.10.4",
-    "@martianwallet/aptos-wallet-adapter": "^0.0.3",
-    "@nightlylabs/aptos-wallet-adapter-plugin": "0.2.12",
+    "@haechi-labs/face-sdk": "^1.10.8",
+    "@haechi-labs/face-types": "^1.10.8",
+    "@martianwallet/aptos-wallet-adapter": "^0.0.4",
+    "@nightlylabs/aptos-wallet-adapter-plugin": "^0.2.12",
     "@openblockhq/aptos-wallet-adapter": "^0.1.5",
     "@pontem/wallet-adapter-plugin": "^0.2.0",
     "@rise-wallet/wallet-adapter": "^0.1.2",
     "@tp-lab/aptos-wallet-adapter": "^1.0.1",
     "@trustwallet/aptos-wallet-adapter": "^0.1.6",
-    "@welldone-studio/aptos-wallet-adapter": "^0.1.2",
+    "@welldone-studio/aptos-wallet-adapter": "^0.1.4",
     "antd": "^5.1.2",
-    "aptos": "^1.14.0",
-    "fewcha-plugin-wallet-adapter": "^0.1.2",
+    "aptos": "^1.17.0",
+    "ethers": "^5.7.2",
```

Comment:
This is required by some wallet, I think it's facewallet.  We could just remove it, since it feels messy that a wallet needs a transitive dependency like this.